profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MyNamePronunciationEdit.java

```diff
@@ -203,7 +203,13 @@ protected void onSubmit(AjaxRequestTarget target, Form form) {
                     }
 
                 } else {
-                    formFeedback.setDefaultModel(new ResourceModel("error.profile.save.info.failed"));
+                    UserProfile userProfile = (UserProfile) form.getModelObject();
+                    
+                    if (userProfile.getPhoneticPronunciation()==null) {
+                        formFeedback.setDefaultModel(new ResourceModel("error.profile.save.phonetic.alert"));
```

Comment:
What about this?
String errorMessage = userProfile.getPhoneticPronunciation() == null ? "error.profile.save.phonetic.alert" : "error.profile.save.info.failed";
formFeedback.setDefaultModel(new ResourceModel(errorMessage));